### ISSUE TITLE
New Site: comments on pages?

### ISSUE THREAD

**Issue by hzoo** [babel/website#1554]
I think https://github.com/orta/gh-commentify would be great to use instead of disqus or anything else. Also I don't ever go on our discourse atm so not sure what we should do about that. cc @orta 

And maybe we need a blurb about joining slack/twitter instead of just a link.

**Comment by xtuc** [comment]
I don't go on disqus either, and to be honest I don't think I'll use another solution.

I would be great to have them on Slack. We could redirect them to the Slack inviter.

**Comment by hzoo** [comment]
I was thinking ^ would work out ok because at least it would be on github.

**Comment by orta** [comment]
Yeah, I try to avoid using disqus, comments end up silo'd away. You get "real" admin tools  set up with default flow etc, the tricky bit is keeping the pages mapped to an issue. This is easy for our blog, where you opt in to having comments on a post, but trickier on a big OSS project. 

You might get more value from https://utteranc.es - which does a lot of that kind of work for you also

**Comment by xtuc** [comment]
I agree, GitHub would be great for that. 

I'm wondering if that would work with GitHub discussions? I'm not sure they are public, right?

**Comment by orta** [comment]
They're not, they wouldn't be useful here at all I'm afraid
